napari_animation/animation.py

```diff
@@ -26,12 +27,12 @@ class Animation:
     def __init__(self, viewer):
         self.viewer = viewer
 
-        self.key_frames = []
+        self.key_frames = EventedList()
         self.frame = -1
```

Comment:
In long run we could make `frame` be evented too, so you could connect to an event whenever this changed? Would that help? Probably something we can save for a new PR though